pkg/network/node/ovscontroller.go

```diff
@@ -81,7 +81,7 @@ func (oc *ovsController) SetupOVS(clusterNetworkCIDR []string, serviceNetworkCID
 		return err
 	}
 	_ = oc.ovs.DeletePort(Tun0)
-	_, err = oc.ovs.AddPort(Tun0, 2, "type=internal")
+	_, err = oc.ovs.AddPort(Tun0, 2, "type=internal", fmt.Sprintf("mtu_request=%d", mtu))
```

Comment:
Tested this patch on dind and the fix is working.

Reply:
Looks like we did try to set the MTU on tun0 before in here:
https://github.com/knobunc/origin/blob/3649e14ebf846fe242c793c3d6f6635870c64841/pkg/network/node/sdn_controller.go#L169-L171
Don't know why that did not work? With this new change, we can remove those 3 lines.

Reply:
No, the behavior is confusing.  Aaron Conole is looking at the OVS behavior to try to understand why it changes (and it may be a bug in OVS).  I have no problem removing those lines, but I wasn't sure if we wanted to keep my change long-term since it feels like a work-around.

Reply:
ah, probably OVS doesn't let you adjust the MTU with the ordinary APIs for some reason, but we didn't notice that we were doing it wrong because as long as there's a pod on the node tun0 ends up with the right MTU anyway.